static/js/compose_ui.js

```diff
@@ -39,14 +39,37 @@ exports.smart_insert = function (textarea, syntax) {
     textarea.trigger("autosize.resize");
 };
 
-exports.insert_syntax_and_focus = function (syntax, textarea) {
+exports.insert_syntax_and_focus = function (syntax, {textarea, block_format = false} = {}) {
```

Comment:
We don't allow ES6 syntax in the codebase.

Reply:
Ok. I've changed the syntax in new commit